presto-tpch/src/main/java/io/prestosql/plugin/tpch/TpchConnectorFactory.java

```diff
@@ -125,6 +131,22 @@ public ConnectorNodePartitioningProvider getNodePartitioningProvider()
             {
                 return new TpchNodePartitioningProvider(nodeManager, splitsPerNode);
             }
+
+            @Override
+            public List<PropertyMetadata<?>> getSessionProperties()
+            {
+                return ImmutableList.of(
```

Comment:
There should be corresponding feature config. Maybe let's use feature config instead of session properties?